SSA/Projects/InstCombine/InstCombineBase.lean

```diff
@@ -13,6 +13,10 @@ inductive BaseType
   | bitvec (w : Nat) : BaseType
   deriving DecidableEq
 
+instance : Repr BaseType where 
+  reprPrec 
+    | .bitvec w, _ => "bitvec<" ++ repr w ++">"
```

Comment:
I would either go for the Lean version `"Bitvec " ++ repr w` or the MLIR version `"i" ++ repr w`, but this looks like some C++ inspired stuff
```suggestion
    | .bitvec w, _ => "i" ++ repr w ++">"
```